templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -349,6 +349,14 @@ Parameters:
       - 'Amazon Aurora PostgreSQL'
     ConstraintDescription: Must be 'Amazon Aurora PostgreSQL' or 'PostgreSQL'.
     Type: String
+  DBEngineVersion:
+    Default: 9
+    AllowedValues:
+      - 9
+      - 10
+      - 11
```

Comment:
Okay, so... Turns out CFN template mappings can only be alphanumeric, meaning the options presented to the user will have to be:

    9
    10
    11